### DISCUSSION TITLE
Why it exits after idle

### DISCUSSION THREAD

**Discussion by dezza** [drakkan/sftpgo#1483] [Q&A]
It seems the container exits (`2.5.5` and `2.5.6` tested) after some time idling.

`{"level":"debug","time":"2023-12-12T16:32:31.521","sender":"service","message":"Received interrupt request"}
{"level":"debug","time":"2023-12-12T16:32:31.522","sender":"plugins","message":"cleanup"}
{"level":"info","time":"2023-12-12T16:32:31.522","sender":"common","message":"number of connections with active transfers: 0"}`

I've tried to set

` -e SFTPGO_GRACE_TIME=999999999999999999 -e SFTPGO_COMMON__IDLE_TIMEOUT=999999999999999999` but it doesn't change anything.

Is this intentional?

**Comment by rhallier** [comment]
Hi,
I have exactly the same issue, havent figured out to solve it yet...

**Reply by dezza** [reply]
@rhallier for me it was trivial but not initially obvious. I would recommend you to start process/container with strace it might be as simple as a permission issue or a directory that doesn't exist.

**Comment by rhallier** [comment]
Well, I didnt pay enough attention to your initial issue.
Actually when I'm logged in the web application (sftpgo ui), after 30 secs of inactivity I'm logged out. And I havent found out yet how to set the session timeout ? Have you got the same pb ?
Regards
